File: problems/prob19/prob19.py
def key_with_max_val(dict): 
	values=list(dict.values())
	keys=list(dict.keys())
	return keys[values.index(max(values))]

people = {"mike":30, "jack":22, "frank":16, "karl":19, "matt": 37}
print "The oldest person is " + str(key_with_max_val(people))

File: problems/prob19/prob19.js
function key_with_max_val(hash) {
	var max = -Infinity;
	for(x in hash) {
    	if(hash[x] > max) {
    		max = hash[x];
    	}	
	}
	return max
}


var people = {"mike":27, "jack":22, "frank":16, "karl":19, "matt": 37}
document.write("The oldest person is " + (key_with_max_val(people)))
